<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Экспорт данных</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Экспорт</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QFrame" name="frameSrc">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="editHostSrc">
              <property name="toolTip">
               <string>host</string>
              </property>
              <property name="placeholderText">
               <string>host</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editUserSrc">
              <property name="toolTip">
               <string>user</string>
              </property>
              <property name="placeholderText">
               <string>user</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editPasswordSrc">
              <property name="toolTip">
               <string>password</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btConnectSrc">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboDbSrc">
              <property name="toolTip">
               <string>db</string>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboSchemaSrc">
              <property name="toolTip">
               <string>schema</string>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboTableSrc">
              <property name="toolTip">
               <string>table</string>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listFieldsSrc"/>
            </item>
            <item>
             <widget class="QLineEdit" name="editWhere">
              <property name="toolTip">
               <string>Where</string>
              </property>
              <property name="placeholderText">
               <string>WHERE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btPrepare">
              <property name="text">
               <string>Prepare</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QTableWidget" name="tableData">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QFrame" name="frameDsc">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="editHostDsc">
              <property name="toolTip">
               <string>host</string>
              </property>
              <property name="placeholderText">
               <string>host</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editUserDsc">
              <property name="toolTip">
               <string>user</string>
              </property>
              <property name="placeholderText">
               <string>user</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editPasswordDsc">
              <property name="toolTip">
               <string>password</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btConnectDsc">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboDbDsc">
              <property name="toolTip">
               <string>db</string>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboSchemaDsc">
              <property name="toolTip">
               <string>schema</string>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboTableDsc">
              <property name="toolTip">
               <string>table</string>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listFieldsDsc"/>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Очистка перед экспортом</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btExport">
              <property name="text">
               <string>Export</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Скрипт</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="editScriptHost">
              <property name="placeholderText">
               <string>host</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editScriptUser">
              <property name="placeholderText">
               <string>user</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editScriptPassword">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btScriptConnect">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>248</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btScriptExecute">
              <property name="text">
               <string>Execute</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btScriptSelectAll">
                 <property name="text">
                  <string>Выбрать все</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btScriptUnselectAll">
                 <property name="text">
                  <string>Сбросить выделение</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listScriptDb"/>
            </item>
           </layout>
          </widget>
          <widget class="QTextEdit" name="editScript"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Функции</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QWidget" name="widgetFunc" native="true">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="editFuncHost">
             <property name="text">
              <string>127.0.0.1</string>
             </property>
             <property name="placeholderText">
              <string>HOST</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editFuncUser">
             <property name="placeholderText">
              <string>USER</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editFuncPassword">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>PASSWORD</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboFuncDb"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboFunc"/>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>324</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btFuncUpdate">
             <property name="text">
              <string>Update</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="editFuncText"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="actionSaveIntoFile"/>
    <addaction name="actionLoadFromFile"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionSaveIntoFile">
   <property name="text">
    <string>Сохранить в файл</string>
   </property>
  </action>
  <action name="actionLoadFromFile">
   <property name="text">
    <string>Загрузить из файла</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
